templates/default/src/pages/send-token.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  Select,
+  Stack,
+  useClipboard,
+  useToast,
+} from "@chakra-ui/react";
+import { useAccount, useActiveChain, useSendTokens } from "graz";
+import type { FormEvent } from "react";
+import { useMemo, useRef, useState } from "react";
+
+const SendToken = () => {
+  const { data } = useAccount();
+  const activeChain = useActiveChain();
+  const transactionHash = useRef("");
+  const { onCopy } = useClipboard(transactionHash.current);
```

Comment:
We can just use `navigator.clipboard.writeText("text");`

Reply:
where I can get the navigator?